packages/contracts/README.md

```diff
@@ -2,6 +2,15 @@
 
 Community directory curator contracts
 
+# Deployments
+
+| **Contract**           | **Address**                                                                                                                    | **Snapshot**                                                                   |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------ |
+| **Goerli**             |                                                                                                                                |                                                                                |
+| Directory              | [`0x2fC419c6Ec940A3689922A845F5067577b2d9846`](https://goerli.etherscan.io/address/0x2fC419c6Ec940A3689922A845F5067577b2d9846) | [`5f0dd58`](https://github.com/status-im/communities-contracts/commit/5f0dd58) |
+| VotingContract         | [`0xCDE984e57cdb88c70b53437cc694345B646371f9`](https://goerli.etherscan.io/address/0xCDE984e57cdb88c70b53437cc694345B646371f9) | [`5f0dd58`](https://github.com/status-im/communities-contracts/commit/5f0dd58) |
+| FeaturedVotingContract | [`0xCe4839946e3e51dd467A75b93FF9d33B408E90d7`](https://goerli.etherscan.io/address/0xCDE984e57cdb88c70b53437cc694345B646371f9) | [`5f0dd58`](https://github.com/status-im/communities-contracts/commit/5f0dd58) |
```

Comment:
Is that also reflected in the codebase?

Reply:
Meaning?

Reply:
https://github.com/status-im/community-dapp/blob/7ac15c6dd69541134ebc67943c853c0d8ca9c897/packages/DApp/src/constants/contracts.ts

Reply:
Ah, no I haven't touched the app. This is just for @jkbktl to test his dapp changes on Goerli.

I expect the DApp to be broken now because the contracts introduced a breaking change.